packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -32,6 +32,17 @@ export const inputBoxStyle = mergeStyles({
   }
 });
 
+/**
+ *
+ * @private
+ */
+export const textContainerStyle: IStyle = {
+  alignSelf: 'center',
+  position: 'relative',
+  width: '100%',
+  maxWidth: '41.25rem'
```

Comment:
Does it need this maxWidth? Can it just fill its container and have the container set the desired width?

Reply:
i.e. I see its the same width as the message thread:
![image](https://user-images.githubusercontent.com/2684369/135661684-5eca072b-6150-42f0-910a-31b6d1b63fe1.png)

Reply:
These are changes suggested by alexp